### DISCUSSION TITLE
Plugin example syntax different in react-markdown GitHub readme and react-markdown node_module readme

### DISCUSSION THREAD

**Discussion by Typewritergoblin** [remarkjs/remark#736] [General] [answered]
This example is given on the [react-markdown Github page](https://github.com/remarkjs/react-markdown/blob/d36048a6cb4d03a860d8cf1d209b1cdd2df837f8/readme.md):

```
import React from 'react'
import ReactMarkdown from 'react-markdown'
import {render} from 'react-dom'
import gfm from 'remark-gfm'

const markdown = `A paragraph with *emphasis* and **strong importance**.

> A block quote with ~strikethrough~ and a URL: https://reactjs.org.

* Lists
* [ ] todo
* [x] done

A table:

| a | b |
| - | - |
`

render(<ReactMarkdown remarkPlugins={[gfm]} children={markdown} />, document.body)
```
However, the node module read.md file uses this syntax:

```
render(<ReactMarkdown plugins={[gfm]} children={markdown} />, document.body)
```

The second example is the one that worked for me.

**Comment by ChristianMurphy** [comment, marked as answer]
The documentation in the current readme is for react-markdown version 6, which uses `remarkPlugins` and `rehypePlugins`.
The second example you are showing appears to be react-markdown version 5, which uses `plugins`.
These changes are noted in the changelog https://github.com/remarkjs/react-markdown/blob/main/changelog.md#600---2021-04-15

**Comment by Typewritergoblin** [comment]
Thanks!
